### PR TITLE
Polymorphism: adapt documentation

### PR DESCRIPTION
**Pull request by Polo2**
Add support about discriminator

This commit add a small paragraph about discriminator support
in polymorphism. It should be completed by a more complete blog post
that I haver to write.

Note:
Related API https://bump.sh/bump/doc/petstore-alternatives#operation-post-pets-body-body-alternative
has already been updated with a `discriminator` field, of course URL is valid.

### CODE REVIEW

**Review by jayk-u — COMMENTED** [review]
Not sure about the gif, it seems a bit too rich to me.
Could you just focus on the main column (I don't feel that left and right column have any value for the gif, it would increase visibility)

**Comment by Polo2** [comment]
> Not sure about the gif, it seems a bit too rich to me. Could you just focus on the main column (I don't feel that left and right column have any value for the gif, it would increase visibility)

Sure !
My idea here was to reproduce the same gif with the new discriminator information, but as you noticed they are obviously too much information. I'll record a new one, and while here I'll favor the white mode (as we can see, discriminator purple on dark mode is not really visible, i'll notice our 🧑‍🎨  team about this).

File: docs/help/specifications-support/polymorphism.md [hunk 1]
```diff
@@ -11,7 +12,18 @@ Here's an [example of polymorphism](https://bump.sh/bump/doc/petstore-alternativ
...
+# Discriminator
+
+Both OpenAPI and AsyncAPI specifications provide support of a `discriminator` field,
+to easily identify which schema is used.
+
+In our generated API documentation, this `discriminator` property is identified
+with a specific flag, and allowed value is explicit.
+
+In example provided below:
+* `type` is the discriminator property between alternatives 'Human' or 'Pet'. Value has to be either `'human'`, either `'pet'`.
```
Comment (paulRbr) [inline docs/help/specifications-support/polymorphism.md:28]:
```suggestion
* `type` is the discriminator property between alternatives 'Human' or 'Pet'. Value has to be either `'human'` or `'pet'`.
```

File: docs/help/specifications-support/polymorphism.md [hunk 2]
```diff
@@ -11,7 +12,18 @@ Here's an [example of polymorphism](https://bump.sh/bump/doc/petstore-alternativ
...
+* `kind` is the discriminator property between alternatives 'Dog' or 'Cat'. Value has to be either 🐕, either 🐈.
```
Comment (paulRbr) [inline docs/help/specifications-support/polymorphism.md:29]:
```suggestion
* `kind` is the discriminator property between alternatives 'Dog' or 'Cat'. Value has to be either `🐕` or `🐈`.
```

**Comment by Polo2** [comment]
Thanks both of you for your feedbacks.

I've adapted text and gif following your suggestions (and maybe I added a new pet, maybe).

May I ask you a second review, and maybe an approval here 🙏

**Pull request merged**